docs/development.md

```diff
@@ -147,16 +147,16 @@ A more convenient way to run checks is to via the scripts in the [check/](https:
 
 The above scripts are convenient and reasonably fast, but they often won't exactly match the results computed by the continuous integration builds run on travis.
 For example, you may be running an older version of `pylint` or `numpy`.
-In order to run a check that is significantly more likely to agree with the travis builds, you can use the [continuous-integration/check.sh](https://github.com/quantumlib/Cirq/blob/master/continuous-integration/check.sh) script:
+In order to run a check that is significantly more likely to agree with the travis builds, you can use the [check/all](https://github.com/quantumlib/Cirq/blob/master/check/all) script:
 
 ```bash
-./continuous-integration/check.sh
+./check/all
 ```
 
 This script will create (temporary) virtual environments, do a fresh install of all relevant dependencies and run all relevant checks within those clean environments.
```

Comment:
> This script will create (temporary) virtual environments

Is this still true? I did a quick check of `all` and i don't see anything that may point to new envs being spun up.

Reply:
You are correct.  Updated.